Exercise_1_solution.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[2]:
+
+
+# Implementation of Stack using Arrays
+
+class myStack:
```

Comment:
This implementation does not satisfy the requirements of the assignment. Please implement stack using top variable